### PR TITLE
Update exercise_5.py_Yingda Zhao_Michael

### PR DESCRIPTION
**Pull request by CodeArt98702**

### CODE REVIEW

File: class02/exercise_5.py [hunk 1]
```diff
@@ -18,9 +19,16 @@
 # >>> Number of odd numbers : 5
 
 # code up your solution here
-
-
-
+even = 0
+odd = 0
+numbers = list(int(num) for num in input("Enter the list numbers separated by ',': ").strip("(").strip(")").strip().split(","))
```
Comment (emma-oc) [inline class02/exercise_5.py:24]: this line is throwing out an error, function seems to work fine

File: class02/exercise_5.py [hunk 2]
```diff
@@ -49,25 +62,36 @@
...
 
-
+list1 = ["Fizz" if i%3 == 0 and i%5 != 0 else "Buzz" if i%3 != 0 and i%5 == 0 else "FizzBuzz" if i%3 ==0 and i%5 == 0 else i for i in range(51)]
+print(list1)
 
 
 # 7. Pcik one of the questions above and use while loop for a different solution
 
 # code up your solution here
+num = 1500
```
Comment (emma-oc) [inline class02/exercise_5.py:90]: current code is an infinite loop, please fix it.

**Comment by emma-oc** [comment]
Fix a  few minor things for Ex5, I'll look for Ex6 and 7 in a separate PR.

**Comment by emma-oc** [comment]
Also, no need for `;` at the end.

File: class02/exercise_5.py (hunk 2 above)
Comment (Michael0801949) [reply class02/exercise_5.py:90]: num = 1500
while num >= 1500 and num <=2700:
  if num%7 == 0 and num%5 == 0:
    print(num)
    num = num + 1
  else:
    num = num +1
    continue

Comment (CodeArt98702) [reply class02/exercise_5.py:90]: I added "num =num+1" after "else:", problem solved

**Pull request closed without merging**